### ISSUE TITLE
successful download-file returns false in bash

### ISSUE THREAD

**Issue by rashkov** [Sude-/lgogdownloader#167]
As you can see below, a successful download has a return code of 1, which means false in a Linux shell. Unfortunately this is breaking the use of lgogdownloader as a registered downloader for Arch linux packages, because the scripts rely on a return value of 0 to signal success. There are probably other Linux scripting issues in other distributions that arise from this. Can this please be fixed? Thanks for your hard work on this very useful tool.

```
-> % lgogdownloader --download-file heroes_of_might_and_magic_3_complete_edition/en1installer0
Getting game names (1/1) 17 / 17
Downloading: ./setup_heroes_of_might_and_magic_3_complete_4.0_(28740).exe
100% ▕██████████████████████████▏ 0.78/0.78MB @ 803.24kB/s ETA: 0s
-> % echo $?
1
-> % false
-> % echo $?
1

```

**Comment by Sude-** [comment]
Apparently when I rewrote Downloader::downloadFileWithId to use Galaxy API in a884e8c I accidentally reversed the return code meaning.

This is now fixed in aa29adf

**Comment by rashkov** [comment]
Thanks very much!!
